libs/src/api/user.js

```diff
@@ -683,6 +684,47 @@ class Users extends Base {
   _cleanUserMetadata (metadata) {
     return pick(metadata, USER_PROPS.concat('user_id'))
   }
+
+  async _updateReplicaSet (userId, metadata) {
```

Comment:
can you throw a function comment + for explicitness maybe call this `_updateReplicaSetOnChain()`

Reply:
also nit - would be cleaner/more explicit to have this function just accept the endpoint field instead of entire metadata object

Reply:
totally agreed on both counts